web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -330,20 +332,206 @@ Once we confirm everything works well locally, we will apply the same changes to
 
 You can follow these steps to migrate to the new auth system (assuming you already migrated the project structure to 0.12, as described [above](#migrating-your-project-to-the-new-structure)):
 
+1. If you used `getUserFieldsFn` and/or `additionalSignupFields`, you will need to migrate them to the new `userSignupFields` field.
+
+  In Wasp 0.11.X, you could define a `getUserFieldsFn` to specify extra fields that would get saved to the `User` when using Google or GitHub to sign up.
+    
+  You could also define `additionalSignupFields` to specify extra fields for the Email or Username & Password signup process.
+
+  In 0.12.X, we unified these two concepts into the `userSignupFields` field.
```

Comment:
```suggestion
1. **Migrate `getUserFields` and/or `additionalSignupFields` (in the `main.wasp` file) to the new `userSignupFields` field.**  
  If you are not using them, you can skip this step.

  In Wasp 0.11.X, you could define a `getUserFieldsFn` to specify extra fields that would get saved to the `User` when using Google or GitHub to sign up.
  You could also define `additionalSignupFields` to specify extra fields for the Email or Username & Password signup process.
  In 0.12.X, we unified these two concepts into the `userSignupFields` field.
```

Reply:
Might I suggest we use less sentences in parentheses: for me it makes me shift focus a lot. I have a feeling that there are multiple different streams of thought then.

Using plain sentences or multiple sentences feels better to me:

> **Migrate `getUserFields` and/or `additionalSignupFields` (in the `main.wasp` file) to the new `userSignupFields` field.**  

could become

> **Migrate `getUserFields` and/or `additionalSignupFields` in the `main.wasp` file to the new `userSignupFields` field.**  

or 

> **Migrate `getUserFields` and/or `additionalSignupFields` to the new `userSignupFields` field.** 

If it's obvious from the context, maybe, I'm not sure.

Reply:
We should mention `main.wasp` though, they can't know that. But I am ok with the option where it is not in parenthesses, I get what you mean with breaking the flow.

So you want to go with the second thing then:

Migrate getUserFields and/or additionalSignupFields in the main.wasp file to the new userSignupFields field.